webapp/components/test-file-results.js

```diff
@@ -43,8 +43,8 @@ class TestFileResults extends WPTFlags(LoadingState(PathInfo(
     </style>
 
     <div class="right">
-      <label class="pad">Expand</label>
-      <paper-toggle-button class="pad" checked="{{isVerbose}}">
+      <paper-toggle-button checked="{{isVerbose}}">
+        Expand
```

Comment:
This seems like a chance to better explain what this toggle does. `Show Details`? `Detailed Output`? `Expand Failures`?

Reply:
```suggestion
        Show Details
```

"Show Details" sounds good!